main.nf

```diff
@@ -554,17 +547,10 @@ process wgdOrthologs {
     cd $PWD
     echo "NF internal work directory for [wgdOrthologs (${task.index})] process:\n\$processDir\n" > $logs_folder/wgd_orthologs_${species1}_${species2}.log
 
-    if [ -z \${NSLOTS+x} ]; then
-        nThreads=$params.nThreadsOrthologs
```

Comment:
Did you also remove the parameter `nThreadsOrthologs`? Or do we still need it for something else?